js/chirp.js

```diff
@@ -1,5 +1,11 @@
 function chirp(n){
   // FIX ME
+  if (n === 0) {
+    return '';
+  }
+  
+  chirp(--n);
+  return 'chirp ' + chirp(n);
```

Comment:
I'm not sure how this is working since you're calling chirp without changing the input. You should only be calling chirp once within this function.

Reply:
the --n in the chirp(--n) invocation is whats decrementing the input (and then returning out when n gets down to 0)... should I change the way it works?

Reply:
I was trying to refer to the fact that you're calling the chirp function twice recursively. You really only need to call it on line 8 and that one should have the decremented n value being passed in.